sale_input_barcode/tests/test_.py

```diff
@@ -0,0 +1,24 @@
+# © 2022 David BEAL @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests import TransactionCase
+
+
+class Test(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+    def test_input_line(self):
+        self.env = self.env(context=dict(self.env.context, tracking_disable=True))
+        barcode = "01034009338167591714050010B04059A\x1d310500638515140501"
+        product = self.env["product.product"].create(
+            {
+                "name": "barcode test",
+                "barcode": barcode[2:16],
+            }
+        )
+        assert product
```

Comment:
It's better to use self.assertTrue() as we are in Odoo tests context (and not pure python one).

Reply:
I really don't share your opinion. I'll ask you ...

Reply:
While we do not use https://github.com/camptocamp/pytest-odoo I understand you ask me to avoid assert

Reply:
Indeed.

Reply:
Do you think it could be interesting to promote pytest-odoo towards Oca ?
Are you in favor ?

Reply:
We are currently having the debate internally. I personally use it for some weeks. That would imply a lot of changes. One of them is it will break tests flows for module changing some standard behaviours. That will lead to repositories numbers growing as we will need to identify groups of modules more precisely.

We can maybe open an issue on maintainers repo

Reply:
I don't want imply such changes with my own person as I have not the capacity to be involved to solve them.
Thanks for the information.